client/app/src/accounts/account.controller.js

```diff
@@ -434,6 +481,198 @@
       gettextCatalog.setCurrentLanguage(self.language)
     }
 
+    self.selectAllDateFormats = function () {
+      return DATE_FORMATS
+    }
+
+    self.setDateFormat = function () {
+      function getDateFormat (value) {
+        for (var prop in DATE_FORMATS) {
+          if (DATE_FORMATS.hasOwnProperty(prop)) {
+            if (DATE_FORMATS[prop] === value) {
+              return prop
+            }
+          }
+        }
+      }
+
+      self.dateFormat = getDateFormat(this.selectedDateFormat)
+      storageService.set('dateFormat', self.dateFormat)
+
+      refreshCurrentAccountView()
+    }
+
+    self.getMarketInfo = function (symbol) {
```

Comment:
I'm not too sure what's happened here @fluorine. Code is being added that was removed in PR https://github.com/ArkEcosystem/ark-desktop/pull/461

Reply:
Basically all Market stuff was removed because it's not used